docs/docs/rest-api/public/api/v2/types/volumes.raml

```diff
@@ -7,42 +7,29 @@ uses:
   secrets: secrets.raml
 
 types:
-  HostVolume:
+  ExternalVolumeInfo:
     type: object
     properties:
-      name:
-        type: strings.Name
-        description: The name of the volume to reference.
-      host:
+      size?:
+        type: integer
+        format: int64
+        minimum: 0
+        description: The size of the external volume in GiB
```

Comment:
In my view GiB looks a bit coarse. What do you think about MiBs?

Reply:
Why? I think it is what it is. Changing it would mean a breaking change.

Reply:
Oh geez. 🤦‍♂️ ExternalVolume size is expressed in GiB, PersistentVolume size in MB.

Reply:
there is a JIRA that's tracking this problem. It's consistently a source of confusion for everyone and should be fixed. https://jira.mesosphere.com/browse/MARATHON_EE-1666